ci-operator/step-registry/edge-nodes/provision/day2-localzone-nodes/edge-nodes-provision-day2-localzone-nodes-chain.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "edge-nodes/provision/day2-localzone-nodes/edge-nodes-provision-day2-localzone-nodes-chain.yaml",
```

Comment:
`edge-nodes` indicates they are for localzone nodes?

Reply:
`edge-nodes` includes `local-zone` and `wavelength-zone` nodes, `wavelength-zone` will be fully supported in the future.

Reply:
How abut rename `edge-nodes/provision/day2-localzone-nodes/` to `edge-nodes/provision/localzone/post/` or  `edge-nodes/provision/localzone/day2/`

Reply:
`edge-nodes/provision/localzone/day2/` looks good to me, I will use this.